index.html

```diff
@@ -220,11 +247,11 @@ <h2 class="social order2">LET'S CONNECT</h2>
 
                     </div>
                 </div>
-                <div class="container">
+                <div class="container1">
                     <span class="lang">Frameworks</span>
                     <i class="fa-solid fa-angle-right btn1"></i>
                 </div>
-                <div class="container">
+                <div class="container2">
                     <span class="lang">Skills</span>
                     <i class="fa-solid fa-angle-right btn1"></i>
                 </div>
```

Comment:
- [ ] Please fix the hover effect on `<input type="submit" class="submit-btn" value="Get in Touch">` inside your form , currently it is like this :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206868411-73ef4455-5e23-4329-b648-95e5c706b1f9.png)